herbert_core/utilities/classes/@sigvar/sigvar.m

```diff
@@ -53,354 +52,222 @@
             %           Elements are 0 (mask) or 1 (keep)
             %           If an empty array, then assumed all points are retained
             %           If scalar (and signal non-empty) then all elements assumed to be same
-            
+
             narg = numel(varargin);
-            if narg>=1
-                if numel(varargin)==1 && isa(varargin{1},'sigvar')
+            if narg == 0
+                return;
+            end
+            if numel(varargin)==1
```

Comment:
because `nargin` is Matlab build-in function and this one returns similar (do not know, if it worth it, but probably did minimal modifications to existing code.)